frontend/src/styles/layouts.scss

```diff
@@ -0,0 +1,26 @@
+@import "~/styles/variables.scss";
+
+$desktop: 1024px;
+
+@mixin mobile {
+  @media screen and (max-width: $desktop - 1px) {
+    @content;
+  }
+}
+
+@mixin desktop {
+  @media screen and (min-width: $desktop) {
+    @content;
+  }
+}
+
+@mixin container {
```

Comment:
This mixin's name is a bit confusing, is it a page container, a main content container, something more generic?

Reply:
I am going to improve this later, to copy bunch of codes from bulma.